{
	"header": {
		"line": "CVE-2020-28638 (ask_password in Tomb 2.0 through 2.7 returns a warning when pinentry-c ...)",
		"name": "CVE-2020-28638",
		"description": "(ask_password in Tomb 2.0 through 2.7 returns a warning when pinentry-c ...)"
	},
	"annotations": [
		{
			"line": "\t- tomb 2.8+dfsg1-1 (bug #974719; bug #975084)",
			"type": "package",
			"package": "tomb",
			"kind": "fixed",
			"version": "2.8+dfsg1-1",
			"flags": [
				{
					"bug": 974719
				},
				{
					"bug": 975084
				}
			]
		},
		{
			"line": "\t[buster] - tomb <not-affected> (Vulnerability introduced later)",
			"type": "package",
			"release": "buster",
			"package": "tomb",
			"kind": "not-affected",
			"description": "Vulnerability introduced later"
		},
		{
			"line": "\tNOTE: https://github.com/dyne/Tomb/issues/385",
			"type": "NOTE",
			"description": "https://github.com/dyne/Tomb/issues/385"
		},
		{
			"line": "\tNOTE: Introduced by: https://github.com/dyne/Tomb/commit/477ab204439ddb88d7293d3c35a29e29751feda9 (v2.6)",
			"type": "NOTE",
			"description": "Introduced by: https://github.com/dyne/Tomb/commit/477ab204439ddb88d7293d3c35a29e29751feda9 (v2.6)"
		},
		{
			"line": "\tNOTE: https://github.com/dyne/Tomb/pull/386",
			"type": "NOTE",
			"description": "https://github.com/dyne/Tomb/pull/386"
		},
		{
			"line": "\tNOTE: Attempted to be fixed via: https://github.com/dyne/Tomb/commit/15c894dfb41db3ea3290bdf8f958fd9e3503c4bb",
			"type": "NOTE",
			"description": "Attempted to be fixed via: https://github.com/dyne/Tomb/commit/15c894dfb41db3ea3290bdf8f958fd9e3503c4bb"
		},
		{
			"line": "\tNOTE: which only hides the problem.",
			"type": "NOTE",
			"description": "which only hides the problem."
		},
		{
			"line": "\tNOTE: https://github.com/dyne/Tomb/issues/392",
			"type": "NOTE",
			"description": "https://github.com/dyne/Tomb/issues/392"
		}
	]
}
